### ISSUE TITLE
DICOM headers appear to be wrong

### ISSUE THREAD

**Issue by JoaoPeriquito** [openmiblab/pckg-wezel#82]
DICOM headers from  _View -> Series (headers)_ do not correspond to the right image. It appears to me it always show the DICOM headers of the first image of the series (this is valid for Philips and Siemens data)

**Comment by plaresmedima** [comment]
Yes that is indeed what it is doing. I have been meaning to add a slider at
the bottom so you can navigate through the headers of individual images.
Can you create an issue in wezel if it is not yet there?

**Comment by JoaoPeriquito** [comment]
This is an issue in wezel, isn't it? Did you mean something different?

**Comment by plaresmedima** [comment]
yes all good sorry

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*
